docs/reference/dlm/apis/put-lifecycle.asciidoc

```diff
@@ -0,0 +1,85 @@
+[[dlm-put-lifecycle]]
+=== Set the lifecycle of a data stream
+++++
+<titleabbrev>Put Data Stream Lifecycle</titleabbrev>
+++++
+
+experimental::[]
+
+Sets the lifecycle of the targeted data streams.
+
+[[dlm-put-lifecycle-request]]
+==== {api-request-title}
+
+`PUT _data_stream/_lifecycle`
+`PUT _data_stream/<data-stream>/_lifecycle`
+
+[[dlm-put-lifecycle-prereqs]]
+==== {api-prereq-title}
+
+// TODO Update security
+// * If the {es} {security-features} are enabled, you must have the <role>
+// cluster privilege to use this API. For more information, see
+// <<security-privileges>>.
+
+[[dlm-put-lifecycle-desc]]
+==== {api-description-title}
+
+Sets the lifecycle of the specified data streams. If a data stream does not exist,
+the request fails and returns an error.
+
+[[dlm-put-lifecycle-path-params]]
+==== {api-path-parms-title}
+
+`<data-stream>`::
+(Required, string) Comma-separated list of data streams used to limit the request. Supports wildcards (`*`).
+To target all data streams omit this parameter or use `*` or `_all`.
```

Comment:
This means the `data-stream` is optional

Reply:
I don't understand this comment. The way I see it if we do not support `PUT _data_stream/_lifecycle` then it's required but it can also be a wildcard. Right?

Reply:
Yes, now that we don't support `PUT _data_stream/_lifecycle` this makes sense as is. Thanks for removing support for `PUT _data_stream/_lifecycle`